.gitignore

```diff
@@ -1,34 +1,37 @@
-# Files generated by the interpreter
-__pycache__/
-*.py[cod]
-
-# Environment specific
-.venv
-venv
-.env
-env
-
-# Unittest reports
-.coverage*
-
-# Logs
-*.log
-
-# PyEnv version selector
-.python-version
-
-# Built objects
-*.so
-dist/
-build/
-
-# mypy cache
-.mypy_cache
-
-# IDEs
-# PyCharm
-.idea/
-# VSCode
-.vscode/
-# MacOS
-.DS_Store
+# Files generated by the interpreter
+__pycache__/
+*.py[cod]
+
+# Environment specific
+.venv
+venv
+.env
+env
+
+# Unittest reports
+.coverage*
+
+# Logs
+*.log
+
+# PyEnv version selector
+.python-version
+
+# Built objects
+*.so
+dist/
+build/
+
+# mypy cache
+.mypy_cache
+
+# IDEs
+# PyCharm
+.idea/
+# VSCode
+.vscode/
+# MacOS
+.DS_Store
+
+#added by richard

```

Comment:
Could this become a more detailed comment on what this folder is used for?

Reply:
this was just a junk folder I use to store snippets. feel free to remove if no good or comment as "snippet storage"

Reply:
I'd say we should comment it as "folder for testing things about blessed".